### ISSUE TITLE
Get CBOR tag allocation for CoRIM

### ISSUE THREAD

**Issue by nedmsmith** [ietf-rats-wg/draft-ietf-rats-corim#46]
The TCG Endorsement Architecture spec can be used to justify the need for a block allocation from #6.500 to #6.599 from IANA CBOR tag registry.

**Comment by nedmsmith** [comment]
The request was made for each tag defined within the TCG Endorsement Arch spec. There was push back regarding allocation of a range of tags (not found in a public spec), but the experts would somehow document the intention to avoid allocating from within the requested range if it wasn't corim related.

**Comment by nedmsmith** [comment]
Reached out to IANA experts a third time 2023-02-27.

**Comment by yogeshbdeshpande** [comment]
@nedmsmith : have we not addressed this? Can you please check?

**Comment by nedmsmith** [comment]
> [@nedmsmith](https://github.com/nedmsmith) : have we not addressed this? Can you please check?

Yes. IANA agreed to earmark the range for corim. There is some level of sharing since the TCG spec alerady published some values. An errata for the TCG spec is about to be published, that will return 500 and 502 to IETF CoRIM control.
